x/ibc_query/client/cli/tx.go

```diff
@@ -13,51 +13,44 @@ import (
 	"github.com/cosmos/interchain-queries/x/ibc_query/types"
 )
 
-
 func NewMsgCrossChainQueryCmd() *cobra.Command {
 	cmd := &cobra.Command{
-		Use:   "ibc-query [src-port] [src-channel] [query-path] [query-height] [timeout-timeheight] [timeout-timestamp]",
+		Use:   "query [query-path] [query-height] [chain-id] [timeout-timeheight] [timeout-timestamp]",
```

Comment:
command example:

original cli command
```
interchain-queriesd tx ibc_query ibc-query http://blockchains:27011/cosmos/bank/v1beta1/balances/cosmos184dszdfgrcrpudq7g0f6qx9462gzlptlgq6tzk 1 ibc-1 0-7000 1774856957000000000
```

changed cli command
```
interchain-queriesd tx ibc-query query http://blockchains:27011/cosmos/bank/v1beta1/balances/cosmos184dszdfgrcrpudq7g0f6qx9462gzlptlgq6tzk 1 ibc-1 0-7000 1774856957000000000
```

It looks more good!